client/src/components/BottomSheet/BottomSheet.tsx

```diff
@@ -0,0 +1,64 @@
+import { motion, AnimatePresence } from "framer-motion";
+import Modal from "react-modal";
+
+import { IconButton } from "../Button";
+
+Modal.setAppElement("#root");
+
+type BottomSheetProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  onAddAssessment: () => void;
+  onDeleteCourse: () => void;
+};
+
+const BottomSheet: React.FC<BottomSheetProps> = ({
+  isOpen,
+  onClose,
+  onAddAssessment,
+  onDeleteCourse,
+}) => {
+  return (
+    <AnimatePresence>
+      {isOpen && (
+        <Modal
+          isOpen={true}
+          onRequestClose={onClose}
+          overlayClassName="fixed inset-0 bg-black bg-opacity-20 z-50"
```

Comment:
<img width="307" alt="Screenshot 2023-03-19 at 4 13 45 PM" src="https://user-images.githubusercontent.com/20573623/226212796-dcf8a4d9-d571-4484-90cf-5c8c2bef2a14.png">

Overlay color token is incorrect; overlay also has transitions.

Reply:
Overlay color is `#910035` with opacity `32%`. I believe this is an officially provided token from somewhere and I can't recall anymore.

Reply:
And timing function is `ease-emphasized`

Reply:
Ok... same for #244 ?